ios/FluentUI/Card Nudge/CardNudge.swift

```diff
@@ -178,6 +178,20 @@ public struct CardNudge: View, TokenizedControlView {
         self.state = state
         self.tokenSet = CardNudgeTokenSet(style: { state.style })
     }
+
+    // MARK: Constants
+    private static let iconSize: CGFloat = GlobalTokens.iconSize(.xSmall)
```

Comment:
long term thinking. eventhough these arent overridable shouldn't these definition live in token.swift file still because it should be all magically staticly defined?

Reply:
Y'know, I went back and forth there a lot. We technically take a veeeery minor size hit by putting them in TokenSet because the values then need to be `internal` instead of `private`. That said, I definitely do prefer that approach from a code cleanliness perspective. What do you think?

Reply:
Moving to the proposed solution, which actually _reduces_ package size!